conda/gateways/adapters/ftp.py

```diff
@@ -232,6 +232,43 @@ def build_binary_response(request, data, code):
     return build_response(request, data, code,  None)
 
 
+def get_status_code_from_code_response(code):
+    '''
```

Comment:
I'd lean towards leaving it as-is to make it as similar to the original requests-ftp code as possible, but obviously it's not a big deal either way.